src/index.html

```diff
@@ -7,8 +7,44 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="anonymous"
+    >
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+
+    <header class="header">
+      <a href="#Home">
+        <img
+          src="/images/logo.png"
+          alt="MoYo_Logo"
+          class="img__logo"
+        >
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item"><a href="#APPLE" class="nav__link is-active">Apple</a></li>
```

Comment:
fix this in all of the links
```suggestion
          <li class="nav__item">
            <a href="#APPLE" class="nav__link is-active">
              Apple
            </a>
          </li>
```